### DISCUSSION TITLE
simulate/render jsonl files on computer?

### DISCUSSION THREAD

**Discussion by scribblrsam** [HASwitchPlate/openHASP#200] [Q&A] [answered]
I have seen a couple of the demo screens that have a mouse over the screenshot, is there an easy way to design and view on a computer?

**Comment by dgomes** [comment, marked as answer]
There is no "design tool" you need always to write the JSONL file

but in order to iterate quicker, you compile openHASP locally on your computer (MacOS, Linux Windows) and run locally.

**Comment by mvturnho** [comment]
I have compiled openhasp for linux, when I start program I get a blank screen
How can I make this load the config.json and the pages.jsonl

![image](https://github.com/HASwitchPlate/openHASP/assets/2076073/f9026ff4-cfc3-42ba-a913-278d2b44b18c)

**Reply by fvanroie** [reply]
On Linux, put the `pages.jsonl` file in path `/home/bob/.local/share/hasp/` and it should load automatically on start.
You can use the `jsonl`, `clearpage` and `run` commands to make changes via MQTT.
